app/src/main/java/ch/epfl/sdp/peakar/database/Database.java

```diff
@@ -15,10 +15,11 @@
 
 
 public class Database {
-
+    public static final String dbAddress = "https://peakar-default-rtdb.europe-west1.firebasedatabase.app/";
```

Comment:
```suggestion
    public static final String DATABASE_ADDRESS= "https://peakar-default-rtdb.europe-west1.firebasedatabase.app/";
```
Should this follow the way we have named all other constant strings.

Reply:
Yes totally, thanks!